synapse/cortex.py

```diff
@@ -1584,6 +1584,22 @@ async def initServiceRuntime(self):
 
         await self._initStormSvcs()
 
+        # # Check for deprecated properties which are unused and unlocked
+        deprs = await self.getDeprLocks()
+        for propname, locked in deprs.items():
+            if locked:
+                continue
+
+            count = 0
+            for layr in self.layers.values():
+                count += await layr.getPropCount(propname)
```

Comment:
Also, we should set `maxsize=1` here to avoid it counting the full number of props

Reply:
Also, you'll need to check (formname, propname) to find any props which don't have a form associated with them.